Models/JsonStat/JsonStatGenerics.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.JsonStat
+{
+    public abstract class JsonStatGenericDataset
```

Comment:
Perhaps rename this entire file, as ["generics" is reserved for other purposes](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/).

Reply:
Agreed.

Options:
- Remove `Generic`
- Use `Base` instead